src/meili/candidate_search.cc

```diff
@@ -130,18 +130,22 @@ CandidateCollector::WithinSquaredDistance(const midgard::PointLL& location,
 
 // Add each road linestring's line segments into grid. Only one side
 // of directed edges is added
-void IndexBin(const baldr::GraphTile& tile,
+void IndexBin(const graph_tile_ptr& tile,
               const int32_t bin_index,
               baldr::GraphReader& reader,
               CandidateGridQuery::grid_t& grid) {
+  if (!tile) {
+    return;
+  }
```

Comment:
there are still a lot of these new `if`s which i get but technically they are logically impossible. id prefer that these become `assert`s

Reply:
Done.